src/ui/shared/Modal.jsx

```
@@ -39,23 +40,31 @@ export const Modal = ({ isOpen, onClose, title, children }) => {
+      className='fixed inset-0 z-50 flex items-center justify-center bg-(--void-black)/90 cursor-pointer p-4'
       onClick={onClose}
     >
       <div
         ref={dialogRef}
-        className='w-full max-w-md border-4 border-(--toxic-green) p-6 bg-(--void-black) shadow-[0_0_25px_var(--toxic-green-glow)] cursor-auto focus:outline-none'
+        className='relative w-full max-w-md border-2 border-(--toxic-green)/50 p-6 bg-(--void-black) shadow-[0_0_25px_var(--toxic-green-glow)] cursor-auto focus:outline-none group'
         role='dialog'
         aria-modal='true'
         tabIndex={-1}
         onClick={e => e.stopPropagation()}
       >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)